pufferd.go

```diff
@@ -72,11 +74,38 @@ func main() {
 	flag.BoolVar(&version, "version", false, "Get the version")
 	flag.BoolVar(&license, "license", false, "View license")
 	flag.BoolVar(&migrate, "migrate", false, "Migrate Scales data to pufferd")
+	flag.BoolVar(&uninstall, "uninstall", false, "Uninstall pufferd")
 	flag.StringVar(&configPath, "config", "config.json", "Path to pufferd config.json")
 	flag.Parse()
 
 	versionString := fmt.Sprintf("pufferd %s (%s %s)", VERSION, BUILDDATE, GITHASH)
 
+	if _, err := os.Stat(configPath); os.IsNotExist(err) {
+                if _, err := os.Stat("/etc/pufferd/config.json"); err == nil {
+                        configPath="/etc/pufferd/config.json"
+		}else{
+			logging.Error("Cannot find a config file!\npufferd could be unstable")
```

Comment:
prefer to just use 2 logging.Error calls instead of adding \n's, it makes reading a bit easier